modules/meta.analysis/inst/magic/initialize_planting.R

```diff
@@ -1,24 +1,138 @@
+  val_2005 <- get_fallback_value(group_name, trait_id = 2005)
+  val_1534 <- get_fallback_value(group_name, trait_id = 1534)
+  val_14 <- get_fallback_value(group_name, trait_id = 14)
+  val_2057 <- get_fallback_value(group_name, trait_id = 2057)
+  val_3115 <- get_fallback_value(group_name, trait_id = 3115)
+  val_3116 <- get_fallback_value(group_name, trait_id = 3116)
+  val_3117 <- get_fallback_value(group_name, trait_id = 3117)
+  val_1055 <- get_fallback_value(group_name, trait_id = 1055)
+  
+  
+  #if (is.na(val_3441)|is.na(val_128)|is.na(val_3450)|is.na(val_2005)|is.na(val_1534))  val_601  <- get_fallback_class_value(group_name, 601)
+  if (is.na(val_3441)) val_3441 <- get_fallback_class_value(group_name, 3441)
+  if (is.na(val_128))  val_128  <- get_fallback_class_value(group_name, 128)
+  if (is.na(val_3450)) val_3450 <- get_fallback_class_value(group_name, 3450)
+  if (is.na(val_2005)) val_2005 <- get_fallback_class_value(group_name, 2005)
+  if (is.na(val_1534)) val_1534 <- get_fallback_class_value(group_name, 1534)
+  if (is.na(val_14)) val_14 <- get_fallback_class_value(group_name, 14)
+  if (is.na(val_2057)) val_2057 <- get_fallback_class_value(group_name, 2057)
+  if (is.na(val_3115)) val_3115 <- get_fallback_class_value(group_name, 3115)
+  if (is.na(val_3116)) val_3116 <- get_fallback_class_value(group_name, 3116)
+  if (is.na(val_3117)) val_3117 <- get_fallback_class_value(group_name, 3117)
+  if (is.na(val_1055)) val_1055 <- get_fallback_class_value(group_name, 1055)
+  
+  #if (is.na(val_3441)|is.na(val_128)|is.na(val_3450)|is.na(val_2005)|is.na(val_1534)) val_601  <- get_fallback_type_value(group_name, 601) 
+  if (is.na(val_3441)) val_3441 <- get_fallback_type_value(group_name, 3441)
+  if (is.na(val_128))  val_128  <- get_fallback_type_value(group_name, 128)
+  if (is.na(val_3450)) val_3450 <- get_fallback_type_value(group_name, 3450)
+  if (is.na(val_2005)) val_2005 <- get_fallback_type_value(group_name, 2005)
+  if (is.na(val_1534)) val_1534 <- get_fallback_type_value(group_name, 1534)
+  if (is.na(val_14)) val_14 <- get_fallback_type_value(group_name, 14)
+  if (is.na(val_2057)) val_2057 <- get_fallback_type_value(group_name, 2057)
+  if (is.na(val_3115)) val_3115 <- get_fallback_type_value(group_name, 3115)
+  if (is.na(val_3116)) val_3116 <- get_fallback_type_value(group_name, 3116)
+  if (is.na(val_3117)) val_3117 <- get_fallback_type_value(group_name, 3117)
+  if (is.na(val_1055)) val_1055 <- get_fallback_type_value("miscellaneous field", 1055)
+  
+  
+  #Variables used to calculate Carbon and Nitrogen content
+  M_stem <- val_128
+  M_Leaf <- val_3441
+  M_fineRoot <- val_2005
+  M_coarseRoot <- val_1534
+  N_leaf.mass <- val_14
+  CN_fineRoot <- val_2057
+  C_leaf.fraction <- 0.47
+  C_stem.fraction <- 0.47
+  C_fineRoot.fraction <- 0.47
+  C_coarseRoot.fraction <- 0.5
+  CN_root <- val_1055
+  CN_stem <- 70 #placeholder
+  SLA <- (val_3115 + val_3116 + val_3117) / 3
```

Comment:
What's going on here? SLA is a single pre-calculated value, not the average of three things